tests/components/ambiclimate/test_config_flow.py

```diff
@@ -57,36 +58,32 @@
     result = await flow.async_step_user()
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["step_id"] == "auth"
-    assert (
-        result["description_placeholders"]["cb_url"]
-        == "https://hass.com/api/ambiclimate"
-    )
+    assert (result["description_placeholders"]["cb_url"] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)